pipelines/_ingredients/MetricsRuntime.yaml

```diff
@@ -62,6 +69,20 @@ tasks:
       connections.metric: DipoleFitTime
       connections.labelName: detectAndMeasure
       target: detectAndMeasure:measurement.run
+  timing_rbClassify:
+    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
+    config:
+      connections.package: meas_transinet
+      connections.metric: RBTransiNetTime
+      connections.labelName: RBTransiNet
+      target: rbClassify.run
+  timing_transformDiaSourceCat:
+    class: lsst.verify.tasks.commonMetrics.TimingMetricTask
+    config:
+      connections.package: ap_association
+      connections.metric: TransformDiaSourceCatalogTime
+      connections.labelName: transformDiaSourceCat
+      target: transformDiaSourceCat.run
```

Comment:
There's already a `timing_transformDiaSrcCat` task below (new lines 93-99). I'd recommend keeping that entry and just changing `connections.metric` to point to the new metric (assuming we keep it; see [my comment on verify_metrics#40](https://github.com/lsst/verify_metrics/pull/40#discussion_r1304947566)), same for `cputiming_transformDiaSrcCat`.

Reply:
Done; should show up in the next commit.